recipe/meta.yaml

```
@@ -35,10 +36,10 @@ test:
     # for tests
     - iso8601 >=1,<2
     - mock >=4,<5
-    - pytest >=6,<7
+    - pytest >=6,<8
     - pytest-asyncio >=0.16,<2
-    - pytest-benchmark >=3.4,<4
-    - pytest-cov >=3,<4
+    - pytest-benchmark >=3.4,<5
+    - pytest-cov >=3,<5
```

https://github.com/graphql-python/graphene/blob/82903263080b3b7f22c2ad84319584d7a3b1a1f6/setup.py#L50
```suggestion
    - pytest-cov >=5,<6
```